sp_client.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency('rest-client', '~> 1.8')
-  gem.add_dependency('oj', '~> 2.18')
+  gem.add_dependency('oj', '~> 3.4')
```

Comment:
Can you just say '>= 2.18' instead?  Gems should give a wider range of possible values if possible.